__tests__/src/global/assetsImages.js

```diff
@@ -0,0 +1,15 @@
+import Images from "../../../src/global/assetsImagesResources";
+
+
+test('Image Assets', () => {
+
+  expect(Images.background).toEqual({"testUri": "../../../src/assets/images/background-gray.jpg"});
+
+});
+
+
+test('Image Assets Logo', () => {
+
+  expect(Images.privateKeyDemo).toBe(1);
```

Comment:
It looks not obvious why this test should pass. Also test name seems to be not in line with actual test body (test called Image Assets Logo, but privateKeyDemo tested).